test/e2e/test/version.go

```diff
@@ -16,7 +16,7 @@ const (
 	// LatestReleasedVersion6x is the latest released version for 6.x
 	LatestReleasedVersion6x = "6.8.20"
 	// LatestReleasedVersion7x is the latest released version for 7.x
-	LatestReleasedVersion7x = "7.16.2"
+	LatestReleasedVersion7x = "7.16.3"
```

Comment:
nit: I would keep version upgrades in dedicated PRs, they might have some side effects, it makes e2e test failures easier to track.

Reply:
Completely understand.  thanks